### DISCUSSION TITLE
Faster scan_interval only for one sensor

### DISCUSSION THREAD

**Discussion by PatrikTrestik** [wills106/homeassistant-solax-modbus#578] [Q&A]
Hi.

I wonder if it could be possible to modify scan_interval only for one sensor.
For most of values is default 15s optimal. But I need to develop PID regulation based on Grid input power.

Value should be read as fas as possible. I'm using WiFi dongle with Modbus TCP so I know that limitation is 1s. It will be ok for my purpouse.

Obviously I can add separate modbus sensor by hand, but why when we have this nice integration. 
BTW: @wills106  Thanks for your great work.

Thanks for any ideas.

**Comment by alienatedsec** [comment]
Which inverter and dongle model?

**Reply by PatrikTrestik** [reply]
I have Solax G4 with new dongle with Modbus TCP.

Sorry for late answer, I replied to email with believe it will arrive here, but it didn't.

**Reply by wills106** [reply]
I'm not going to implement polling at different rates I have covered this here #384 
It will get to the point where you struggle to poll anything else other than the registers around that one you want speeding up.

Edit:
Also having two Modbus Masters connecting to the Inverter will increase the chance of collisions, if the Inverter will even accept the communications at all.
